.ci.yaml

```diff
@@ -44,7 +44,9 @@ platform_properties:
         ]
       device_type: none
       os: Windows-10
-
+# The current android emulator config names can be found here:
+# https://chromium.googlesource.com/chromium/src.git/+/HEAD/tools/android/avd/proto
+# You may use those names for the android_virtual_device version.
```

Comment:
Give it another thought, does it make more sense to move this to the `linux_android_emulator.json` where the package is defined.